universum/modules/vcs/perforce_vcs.py

```diff
@@ -27,6 +30,11 @@
 P4Exception = None
 
 
+class P4stub(ModuleType):  # replace with proper stubs
+    P4Exception: Exception
+    P4: Callable
```

Comment:
I know that this is a class, but then all attributes/methods of a class should also be stubbed.
Should I do that?

Reply:
This is an external code.
We should not cover it, unless it gives us clear benefits.

Reply:
I added these minimal stubs just to stop mypy from complaining (=